bigtable/client/rpc_retry_policy_test.cc

```diff
@@ -13,65 +13,90 @@
 // limitations under the License.
 
 #include "bigtable/client/rpc_retry_policy.h"
+#include "bigtable/client/chrono_literals.h"
 
 #include <gtest/gtest.h>
 #include <chrono>
+#include <thread>
 
-// TODO(coryan) - these are useful when writing tests, consider refactoring.
-// TODO(coryan) - these are generally useful, consider submitting to abseil.io
-namespace bigtable {
-namespace chrono_literals {
-constexpr std::chrono::milliseconds operator "" _ms(unsigned long long ms) {
-  return std::chrono::milliseconds(ms);
+namespace {
+/// Refactor two test cases ...
+void check_limited_time(bigtable::RPCRetryPolicy& tested) {
```

Comment:
If we're following the C++ style guide, this should be `CheckLimitedTime`.

That said, you have a for loop and conditionals, with `EXPECT_*()` depending on a condition... that's not quite recommended by the GoogleTest style guide.

Reply:
Done on the name.
I understand the point about the for loop in a test, I am fine with how it is.